src/main/java/com/dev/movietheatre/model/dto/UserRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package com.dev.movietheatre.model.dto;
+
+public class UserRequestDto {
```

Comment:
Do you need this Dto?

Reply:
Yes, in AuthenticationController , registration method.